core/iwasm/fast-jit/cg/x86-64/jit_codegen_x86_64.cpp

```diff
@@ -352,7 +352,7 @@ cmp_r_and_jmp_label(JitCompContext *cc, x86::Assembler &a,
 
     bool fp_cmp = cc->last_cmp_on_fp;
 
-    bh_assert(!fp_cmp || (fp_cmp && (op == GES)));
+    bh_assert(!fp_cmp || (fp_cmp && (op == GES || op == GTS)));
```

Comment:
Will this op can be GTS? I remember here only GES is introduced by jit_emit_conversion.c, GTS is not required. And if GTS for fp_cmp is used, we should change the handling of `case GTS` below: currently it doesn't check fp_cmp like `case GES`.

Reply:
theoretical speaking. we should support float-point comparison with both `GTS` and `GES`. 
Since we are planning to refractory this part with the new `CMPCC` IR.  I'd like to take this change back.